apps/prairielearn/python/prairielearn/grading_utils.py

```diff
@@ -185,6 +212,9 @@ def determine_score_params(
     score for a particular question. For elements following PrairieLearn
     conventions, the return value can be used as a key/value pair in the
     dictionary passed to an element's Mustache template to display a score badge.
+
+    Returns:
+        A tuple of
```

Comment:
?

Reply:
Resolved in [163dfab](https://github.com/PrairieLearn/PrairieLearn/pull/11758/commits/163dfab708fb88fca0820d6e6f061f20e2cddebe).